release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -3488,3 +3488,22 @@ Now these IP addresses are periodically collected and made available to be reall
 ==== Upgrading
 
 To upgrade an existing {product-title} 4.6 cluster to this latest release, see xref:../updating/updating-cluster-cli.adoc[Updating a cluster using the CLI] for instructions.
+[id="ocp-4-6-56"]
+=== RHBA-2022:0867 - {product-title} 4.6.56 bug fix and security update
+
+Issued: 2022-03-23
+
+{product-title} release 4.6.56, which includes security updates, is now available. The bug fixes that are included in the update are listed in the link:https://access.redhat.com/errata/RHBA-2022:0867[RHBA-2022:0867] advisory. There are no RPM packages for this release. The RPM packages that are included in the update are provided by the link:https://access.redhat.com/errata/RHSA-2022:0866[RHSA-2022:0866] advisory.
```

Comment:
Remove "There are no RPM packages for this release." And can you also remove this line for 4.6.55? This was a carry-over from 4.6.54 when there were no RPM packages for THAT release, lol.

Reply:
Ahh, that makes sense! "There are no RPM packages for this release." has been removed for 4.6.56 and 4.6.55!